File: cpp/funkcje.cpp
/*
 * wejscie-wyjscie.cpp
 */

#include <iostream>
using namespace std;

int dodaj(int a, int b)
{
    return a+b;
}
int odejmij(int a, int b)
{
    return a-b;
}
int pomnoz(int a, int b)
{
    return a*b;
}
float podziel(float a, float b)
{
    return a/b;
}

int main()
{	
	float a,b;
    
    cout<<"Podaj 1 liczbę: ";
    cin>>a;
    cout<<a<<endl;
    
    cout<<"Podaj 2 liczbę: ";
    cin>>b;
    cout<<b<<endl;

    cout<<"Suma liczb a i b wynosi: "<<dodaj(a, b)<<endl;
    cout<<"Roznica liczb a i b wynosi: "<<odejmij(a, b)<<endl;
    cout<<"Iloczyn liczb a i b wynosi: "<<pomnoz(a, b)<<endl;
    cout<<"Iloraz liczb a i b wynosi: "<<podziel(a, b)<<endl;
	
	return 0;
}

File: python/funkcje.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  wejscie-wyjscie.py

def dodaj(a, b):
    return(a + b)
    
def odejmij(a, b):
    return(a - b)
    
def pomnoz(a, b):
    return(a * b)
    
def podziel(a, b):
    return(a / b)
    

def main(args):
    a = int(input("Podaj liczbe 1: "))
    print(a)
    b = int(input("Podaj liczbe 2: "))
    print(b)
    
    print("Suma: ", dodaj(a, b))
    print("Roznica: ", odejmij(a, b))
    print("Iloczyn: ", pomnoz(a, b))
    print("Iloraz: ", podziel(a, b))
    
	
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
